### PR TITLE
SDI-18 other personal contacts page

### PR DESCRIPTION
**Pull request by steverendell**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -148,7 +148,7 @@ workflows:
             - build
       - hmpps/helm_lint:
           name: helm_lint
-      - hmpps/build_multiplatform_docker:
+      - hmpps/build_docker:
```
Comment (steverendell) [inline .circleci/config.yml:151]: removed slow build of arm version as probably not needed locally

File: README.md [hunk 2]
```diff
@@ -67,3 +67,5 @@ Commands to generate types from the apis:
 `npx openapi-typescript https://api-dev.prison.service.justice.gov.uk/v3/api-docs --output server/@types/nomisPrisonerImport/index.d.ts`
 
 `npx openapi-typescript https://prisoner-offender-search-dev.prison.service.justice.gov.uk/v3/api-docs --output server/@types/prisonerSearchImport/index.d.ts`
+
+`npx openapi-typescript https://prisoner-contact-registry-dev.prison.service.justice.gov.uk/v3/api-docs --output server/@types/prisonerContactRegistry/index.d.ts`
```
Comment (steverendell) [inline README.md:71]: switched to generated types for the visits contact api types

File: integration_tests/e2e/otherContacts.cy.ts [hunk 3]
```diff
@@ -82,8 +81,7 @@ context('Other contacts', () => {
       cy.contains('Mobile 07711333444')
       cy.contains('Business 011333444 extension number 777')
 
-      cy.contains('View details').click()
-      cy.get('details').within(() => {
+      cy.get('dl').within(() => {
```
Comment (steverendell) [inline integration_tests/e2e/otherContacts.cy.ts:84]: removed accordion feature at Eddies request

File: integration_tests/mockApis/prisonApi.ts [hunk 4]
```diff
@@ -19,7 +19,7 @@ const stubGetPrisonerContacts = ({ offenderNo, response }: { offenderNo: string;
   stubFor({
     request: {
       method: 'GET',
-      url: `/api/offenders/${offenderNo}/contacts?activeOnly=true`,
+      url: `/api/offenders/${offenderNo}/contacts`,
```
Comment (steverendell) [inline integration_tests/mockApis/prisonApi.ts:22]: according to Andy the active flag cant be trusted so ignore it

File: server/@types/nomisPrisonerImport/index.d.ts [hunk 5]
```diff
@@ -662,6 +662,18 @@ export interface paths {
     /** Can be filtered by participation type and incident type */
```
Comment (steverendell) [inline server/@types/nomisPrisonerImport/index.d.ts:662]: updated this to include Gareths addition of phones in contact endpoint data

File: server/config.ts [hunk 6]
```diff
@@ -83,11 +83,5 @@ export default {
       enabled: get('TOKEN_VERIFICATION_ENABLED', 'false') === 'true',
     },
   },
-  applicationInsights: {
-    url: get('APPLICATION_INSIGHTS_URL', 'http://localhost:8103/applicationinsights'),
-    subscriptId: get('APPINSIGHTS_SUBSCRIPTION_ID', 'subscription', requiredInProduction),
-    component: get('APPINSIGHTS_COMPONENT', 'component', requiredInProduction),
-    resourceGroup: get('APPINSIGHTS_RESOURCE_GROUP', `${get('APPINSIGHTS_COMPONENT', 'component')}-rg`),
-  },
```
Comment (steverendell) [inline server/config.ts:91]: not required

**Review by andymarke — APPROVED** [review]
LGTM

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**